db/migrate/20190221101857_add_last_to_proxy_rules.rb

```diff
@@ -0,0 +1,5 @@
+class AddLastToProxyRules < ActiveRecord::Migration
+  def change
+    add_column :proxy_rules, :last, :boolean, default: false
```

Comment:
This needs a migration to be done by @3scale/operations because it has a NON-NULL default

Reply:
@macejmic do not forget to remove that when this is merged https://github.com/3scale/porta/pull/658